File: src/codes/Users_Manual/stm/gain_1.cpp
//~#include<autd3.hpp>
//~#include<autd3/link/nop.hpp>
//~int main() {
autd3::GainSTM stm(autd3::SamplingConfig(1 * autd3::Hz),
                   {autd3::gain::Null(), autd3::gain::Null()});
//~return 0; }

File: src/codes/Users_Manual/stm/gain_1.cs
~using AUTD3Sharp;
~using AUTD3Sharp.Utils;
~using AUTD3Sharp.Link;
~using AUTD3Sharp.Gain;
~using static AUTD3Sharp.Units;
~using var autd = Controller.Builder([new AUTD3(Point3.Origin)]).Open(Nop.Builder());
var stm = new GainSTM((SamplingConfig)(1u * Hz), [new Null(), new Null()]);